### ISSUE TITLE
Nested institutions in grscicoll

### ISSUE THREAD

**Issue by rukayaj** [gbif/registry#285]
We have historically used the ["O" institution code](https://www.gbif.org/grscicoll/institution/2e961f1c-399e-4aab-92ea-0657722f8dcd) and semi-recently changed this to the ["NHMO" institution code](https://www.gbif.org/grscicoll/institution/ffe44856-4ea4-4443-ab9f-4b3c7581b2bb). We have some datasets and data sources which still reference the old "O" code, and we would like to tidy this up where possible.

However, the Natural History Museum here is not technically an institution in itself. Our actual institution is the University of Oslo (UiO). and has e.g. a [grid identifier](https://www.grid.ac/institutes/grid.5510.1) and [ror identifier](https://ror.org/01xtthb56). Additionally, UiO has several "child" institutions e.g. the [Natural History Museum](https://www.gbif.org/publisher/f314b0b0-e3dc-11d9-8d81-b8a03c50a862) or the [Department of Biosciences](https://www.gbif.org/publisher/6c1dec01-8c82-4341-8584-26c6576d3b69), which are registered as separate GBIF publishers. 

What would you advise here, perhaps we should publish all datasets from these publishers with the same "uio" institution code?

**Comment by dagendresen** [comment]
**O** is used for the University of Oslo > Natural History Museum > **Botanical museum**
**NHMO** is used for University of Oslo > Natural History Museum > **Zoological museum** 
(and NHMO is proposed as the acronym for the entire museum, but I believe that no consensus is reached between botany and zoology "departments"...)

**Comment by dagendresen** [comment]
The Norwegian university museums (and some faculties, and some institutes for biology/geography/...) are registered as separate data publishers in GBIF, however, the ROR and Grid identifies the entire university...

**Comment by dagendresen** [comment]
https://github.com/gbif/registry/issues/282

**Comment by ManonGros** [comment]
Hi @rukayaj @dagendresen
We don't support child institutions but we now have the possibility to add alternative institutions and collection codes. In addition to that, we also have a "merge" feature (which will automatically transfer identifiers and fill in alternative codes to the remaining institution https://github.com/gbif/registry/issues/255).

So depending on how you would like to represent the institutions, you could have each of them represented as (separate) institutions (and mention in the description the relationship with the others) or you could merge some of them and/or add alternative codes. Of course, merging entities can quickly become quite messy so be sure that this is what you need first.

I am leaving the issue open as a reminder (and use case) to explore nested entities.

**Comment by dagendresen** [comment]
I believe we might want to keep the finer level of sub-units and university departments as data publishers (for accreditation, etc) -- and thus not to merge. But a nested hierarchy of organizations would be very useful!! (including counting of occurrences, citations, etc across all child sub-units of a parent organisation).

**Comment by rukayaj** [comment]
It makes sense to me to merge **institutions** up until we find one of them which has a grid/ror identifier. But I think we can have the different departments as separate publishers in GBIF still, right? It looks like organizations/publishers are treated completely separately from GRSciColl institutions, right @ManonGros? Or maybe it's a better idea to keep the two of them mirrored? I don't really have any strong opinions. 

Thanks for adding the list of institutions + identifiers, @dagendresen !

**Comment by ManonGros** [comment]
@dagendresen I can't promise anything but we will take a look at the idea. But @rukayaj is correct, GrSciColl institutions and GBIF organizations/publishers are **separate**.
You could keep separate GBIF organizations and merge the GrSciColl institutions for example.

Keep in mind that without any nesting system, you can already group the data and generate stats on GBIF based on:
- dataset
- publishing organization
- hosting organization
- node
- projectID
- GrSciColl collection
- GrSciColl institution
- network (not for everyone - more on the scope for those later)
- and of course some other data attributes such as recorder, year, etc.

**Comment by dagendresen** [comment]
I am thinking more of the stats that is presented (user-friendly) to the portal users, than stats that we at the GBIF Node CAN generate :-)

**Comment by ManonGros** [comment]
I meant this type of data aggregation/stats:

- dataset: https://www.gbif.org/dataset/d34ed8a4-d3cb-473c-a11c-79c5fec4d649/metrics
- publishing organization: https://www.gbif.org/publisher/f314b0b0-e3dc-11d9-8d81-b8a03c50a862/metrics
- hosting organization: https://www.gbif.org/occurrence/charts?hosting_organization_key=64795dcb-ad74-41b3-95e5-cc52ba754776
- node: https://www.gbif.org/country/NO/publishing
- projectID: https://www.gbif.org/project/6vsEqjdu6IUGaqKK8Wyyme/strengthening-malawis-biodiversity-information-facility#datasets and https://www.gbif.org/occurrence/charts?advanced=1&project_id=BID-AF2017-0385-NAC&occurrence_status=present
- GrSciColl collection: https://www.gbif.org/grscicoll/collection/fa4814fb-7a6a-450d-8099-e01e42d26f3d/metrics
- GrSciColl institution: https://www.gbif.org/grscicoll/institution/2e961f1c-399e-4aab-92ea-0657722f8dcd/metrics
- network (not for everyone - more on the scope for those later): https://www.gbif.org/network/2b7c7b4f-4d4f-40d3-94de-c28b6fa054a6/metrics

**Comment by wouteraddink** [comment]
I would recommend the Natural History Museum getting its own GRID identifier and register in GRID as child organisation below the University of Oslo (UiO).  That way, in a future PID graph, provenance etc. can be linked directly to the Museum (and also still to the University).

**Comment by dagendresen** [comment]
If GRID supports parent-child hierarchy (and self-registration/curration?) maybe we do not need GrSciColl at all :-)

**Comment by rukayaj** [comment]
I just had the exact same thought :) I wonder if GRID lets you specify that this institute holds scientific collections...

**Comment by dagendresen** [comment]
Should we try -- I can try now.

**Comment by wouteraddink** [comment]
Yes you can use the suggest button for that in their site. GRID metadata is very limited and generic so will not cover all community needs for metadata about an institution I think. The museum would be an "Archive" under GRID, with no way to specify that it holds scientific collections.

**Comment by dagendresen** [comment]
## About GRID numbers for "child"-"institutions" such as the University museums

https://gridac.freshdesk.com/support/tickets/68248

> #68248 New record in GRID for the Natural History Museum in Oslo?
> Dag Endresen, reported 14 days ago
> Is it possible to register a new entry for the UiO Natural History Museum as a child under the University of Oslo (https://www.grid.ac/institutes/grid.5510.1)
> 
> The museum is organized as part of the university at the same level as the faculties.
> 
> See also:
> https://github.com/gbif/registry/issues/285
> 
> UiO Natural History Museum:
> https://isni.org/isni/0000000406138625
> https://www.wikidata.org/wiki/Q1840963
> https://www.gbif.org/grscicoll/institution/ffe44856-4ea4-4443-ab9f-4b3c7581b2bb
> https://www.gbif.org/publisher/f314b0b0-e3dc-11d9-8d81-b8a03c50a862
> https://en.wikipedia.org/wiki/Natural_History_Museum_at_the_University_of_Oslo
> Parent institution https://www.grid.ac/institutes/grid.5510.1

> GRID Support , said an hour ago
> Dear Dag Endresen,
> 
> Thank you for your suggestion and for providing us with such detailed information about this organisation and the underlying use case. Unfortunately creating a GRID record for a department like institution within a university does not line up with our current content policies. 
> ​
> We had an internal discussion how to handle this request and if we were able to do an exception, but the negative effect in doing so would be that the University of Oslo in other systems using GRID would be facing content loss, e.g. publications, which than would be assigned to the UiO Natural History Museum. Therefore we currently cannot support the use case behind this request.
> ​
> Best wishes,
> Maren Wöltje
> ​GRID Support Team

> Dag Endresen , said 30 minutes ago
> Cc: wouter.addink@naturalis.nl, helpdesk@gbif.no
> ​May thanks for considering the request so carefully! And for finding a clear guideline! This clear guideline will assist in our planning and current design of catalogs for biodiversity information holding institutions and collection holding museums.
> Thanks!
> Dag
